pilot/pkg/model/push_context.go

```diff
@@ -52,8 +52,8 @@ var _ Metrics = &PushContext{}
 type serviceIndex struct {
 	// privateServices are reachable within the same namespace, with exportTo "."
 	privateByNamespace map[string][]*Service
-	// public are services reachable within the mesh with exportTo "*"
-	public []*Service
+	// Public are services reachable within the mesh with exportTo "*"
+	Public []*Service
```

Comment:
nit: donot export this, we should not allow operating push content directly incase of conflict. Provide a public method.